package.json

```diff
@@ -22,6 +22,7 @@
     "@types/webdriverio": "^4.7.0",
     "chai": "^4.1.0",
     "chai-string": "^1.4.0",
+    "colors": "^1.4.0",
```

Comment:
@marechal-p Why is it needed for this PR? sorry, I could not figure it out

Reply:
https://github.com/eclipse-theia/theia/blob/727b2762945048ea62ca224bfec6882cebecca90/packages/process/src/common/shell-command-builder.spec.ts#L31

Reply:
ok